File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.swift
class Solution {
    func removeStars(_ s: String) -> String {
        var stack: [Character] = []
        
        var arr: Array<Character> = Array(s)
        
        
        for c in s {
            if  c != "*" {
                stack.append(c)
            } else {
                stack.removeLast()
            }
        }

        return String(stack)
    }
}

File: 2390-removing-stars-from-a-string/2390-removing-stars-from-a-string.java
class Solution {
    public String removeStars(String s) {
        Stack<Character> stack = new Stack<>();

        for (int i=0; i<s.length(); i++) {
            if (s.charAt(i) != '*') {
                stack.push(s.charAt(i));
            } else {
                stack.pop();
            }
        }

        String result = stack.stream().map(a -> a.toString()).collect(Collectors.joining(""));
        return result;
    }
}